index.html

```diff
@@ -284,6 +284,47 @@ <h2 style="color:white">History of <strong class="theme_blue">handheld consoles<
xAVdvkAE8wHYDCOGkYFm50xcAEKLszFO1Xdal27EG+c5+ZrYGNbFZMaALwflq+2Cc5e5dlgyGLxtTcGsg3J5QEXkOGr12ARr7wkBd7nxTnRRJ3YlWyJbYGnkSn9AN+9QGbjUOCNfNzH1nAm2qpAeWi4XoHFqYcBdG8xY3QQSAdzOYUOiByqoF4CLumZqV4Dytmvo3AfSFLG7o1xEYXSV9vd40ImW24GELtbXh1r+7Cm4KsHkF3eyC63CTEukcR/R1ysp+piBRi4uBm/hk/DRgnpZRKe4ELImUImuAW9epYWtHoS+0CLYZYp0jGmmynpiQ3U6bNkR2IMY8riszgdUu/r5MVQWZBC6ZnyV6S82lZxW/mAYDRbBBLXQWkgr5R0VkQqh2fLm83g7159CykG8pJ5pPon8krw8joBLlCYLV+3xLFh9a5GCuxQGZnE8xMpP2msiqqSPincaWd6N/TqrV6XAA8hoQDUgLRSL1Y9MFoH5MtltmIQV67muuakUm92YVuXHkJmTU/v4iWthG5lXXJceqoaVz0HdzgRm61X19qQ0q/+qSVPu/TUlIaXnTqI4bU16maCNXxoPxsEalN6OF2wOUGFRjHtXF1QwVhX1eNI8iBDWCy23U/rXTvHVoCOxg4t/G4ssgkEYgCc6lIpahD/whC82A4tZ7xnfIRi6cYBrWeOboVZR1reu8YtbQyGmHAg6I5mTPT0XlOb44Okr/m0Zo6nsBuCZHTxQ4WS4nVh7KoIX3Q2AW06Mr3FMud0yT20QRwQByMZuWLpvy4pz78429HII+zgBpLArJ8OxH2Y6GziucJANTvLqcGdLCk7EowcsIEaZzz3aYaHmyFJNEQj7vQU5AeWfj8edlIlpLU68368sCybPXgEubIjwQY9PEJqCnoVICeKmGuJexfGHn86DvJNw3V2Hb95WC+2puckx1xzl+jWRqZ86YlhsgMQ6sQKE/a3MqV0E/e4viO8kfy0Y2T25OqcnhimGj31gfQw6kmNorbcHHrek+/qE6kmT490eAmEZ+T4MlUU2J00HFkv8z3rGiOFNzYnihu5+D+iQRj0jzmHthiosNkPqnd2+2vPVZxIj/ZnP+vxXGVl7oi7pvshEaZsD488OzBq2zhCTtvaKRt1FKgN+vcLw9mf161jJMfxWY3Vo8pS7RAuiPiY+8rhcsKjvbFHvdoI09O2nosCo3A+HI9mq/V6tZrO4vH+uMVc7+GRBg/uLLmQEmZEGrsByK2NcNcnptYPSpxT6WwNcjxJj0SGImHi9dRJWKPHQJ4WBcoQjajY+Gb6zCxhSrdWkb54JA5QAznHnnokQ5SeC9QIuMP+eMKj1FEvMxf8PLr0SNxdFDgryFgbu9T9RtnggVVahC4xWXVu75swUWCvSB+3xmkc3DA2wpPSHo5JSn6KnRpUdcVvFt1NOpZ0Z3ViCnEws9LQSu/JisITwkRN2rOLDv7YdYr0zq9nm+rqh0X6LKJlgz2Xz5oqtjJwHqa9D89owFjHqQxpe0fMOYv7SP/xCPm6Wa689Ei+Bgq2ShFhCdsQjdIerpXwRfeBPUdT30X648EBcwHaYLzgAmEUSE8SJZhSKaDB93lEWjgndRcLeJu7p67akWrN7xaey4VRKEDPzFCQgNJIJ5KoSNd9yGhn5jHZnISzYbnr0Xg46EZRUp2yqWlRq7t3L9e2cPaz7m6fm48Z+QLds4n0Dq24Z2TDoSS9YTmGv09dJ2GRtQ8fhMtM73nVTqJQm88knUukd9CGC0ca6SNIcDQa0vDQtPS757U6SZzZ2dxsxz2TSO/RHW+85yJ9vmctOz7E+6Z0xTwh7eGKdmE9g7C/dgLzAM7IN6zG3WGhRg5dRmmdh/DOJM/v1vb+4Go1Vdu1FvHwUOckLwt8cWaRxtC6w8nU7FmuYZs0weGbth1YnrMYLZVnP0tA6apWNJxZpFNoYWe4HE/uZ9PVajqdje4el0+dY1cRkxO+lHRfgEiXiCYcxieZZ2h9om7cSxDpkhCRWcksk9Zl+xN14V6MSBcHXh+cWaF70Xr37vLDz5Swey5Pq2x0pXFf68P+YJ8PqUh334xIdx2R8M0HOMfow09NuOzVizSZ6Ge6LvoJG8Xaf6VRGJkgR96rJ+wKXaclhNt/cJdhyReOoX51oD2cdl1zR7j9V+aytyPSZBLtNiX86fMfwmVEpHt1ViWrAqlIUybN3+ivdBBTka5x9XA1oLIq0UYfL1Mt/WZcy1QbZbvuy9WH65TwWxbpBB+v9naYBlypuX71hKXpyC9J9Sm2hxXf5fWBMGFF+st32CP+gRnDBhnqb4UwI6vaJ/C0Pv/DXPbmRLpxm5qlhDBLl1Far5+wKxjYPWGe7hsX6eanq3+Ey96MSIei0tJ+l1z2dno4UPuM2ldBS4v+yWsDVVqCrEY/22kCIKS+9Gt3LSUinSC6fsc6HqFNxnD921vKBRVprof3dC8v36WuJRnDev3bl0qGxCxdtC5xTuuapvSJINjn2eFSImRa+ivOaaU6i4g08qqqjVIbQlGkmwnh9lf2MpwJqvNUrooQimZpT5inS0o7FCp690JARTr1pS++fv6avQxKd7z62PAiVb+sSP8qrAqCulmmbNneawMl3MtZ7wLnJldVlLRe0DGcRxgOfavslPNakYq0mrB4SMcrBiV8qyR8778hvgeINPRvT/Hrq0NqhxWEi56R/NIQkiVu8h6GEnw1VEarDXQMS3sYanTXecxN5QhJFRAJYU1WgO+1g0T2EpGG2lkFy5K+OChFGheeq6UieZ2gPexwAxVXEHtD2pmAEnZXdMfIvGHnm6rXDEpYtx7Gw8FgGG9JwHjeEgBVgRLe755wA5suts9WEHsruPGlS67t9dkKllSMaCHsGdkpsDfkWQkYe5ntL2YVNW5fEMIRs2lNN9z4jNVo6kHUX986bhAEVm+1fPNsAWFnMJ/nbOj7P94k/ge3pjprV2KaDwAAAABJRU5ErkJggg==">Image 9</a></p>
```

Comment:
Ditto, shouldn't this be

Reply:
... a URL to an image.

Reply:
@BeartoothHiker I did these pretty quickly and thought they all linked to an image except for the one that is a source for two images. Please feel free to fix! I don't have a lot more time to work on this today.

Reply:
Thanks Brink.  I'm trying to see where the links come from, they must already be inline.